CHANGELOG.md

```diff
@@ -1,3 +1,137 @@
+## [1.0.0] - 2024-05-29
```

Comment:
**suggestion (documentation):** Consider standardizing the date format throughout the changelog

Some entries use YYYY-MM-DD format while others use YYYY-DD-MM. Consistency would improve readability.